testing/trino-benchto-benchmarks/README.md

```diff
@@ -48,8 +52,10 @@ data-sources:
 environment:
   name: TRINO-DEV
 
-presto:
+trino:
   url: http://localhost:8080
+  username: na
+  password: na
```

Comment:
The password should not be required by default. Maybe change the example user to `benchto` so it's a bit more informative than `na`.